tests/cli/mytestfile.test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+
+function run_test {
+  # the suite name below will only be used when running this file directly, when
+  # running through "test.sh" it must be provided using the "--suite" option.
+  tinitialize "decK-cli test suite" "${BASH_SOURCE[0]}"
+
+  tchapter "deck file openapi2kong"
+
+  ttest "accepts format 'json'" 
+  ./deck file openapi2kong --format json > /dev/null < tests/cli/fixtures/mock-a-rena-oas.yml
+  if [ $? -ne 0 ]; then
+    tfailure "--format=json was not accepted"
+  else
+    tsuccess
+  fi
```

Comment:
The repeated checks would be error prone. Should we consider using something like https://github.com/bats-core/bats-core that has auto-detection?